KOTH/Abudor_II/map.json

```diff
@@ -91,7 +91,8 @@
 		}
 	],
 	"itemremove": [
-		"stone sword", "arrow", "iron chestplate", "leather boots", "golden apple", "dirt", "golden carrot"
+		"stone sword", "bow", "golden apple", "golden carrot", "arrow",
+		"iron chestplate", "leather boots"
```

Comment:
If there's not going to be many itemremoves, it might be better to just fit it all on one line.